examples/evaluator-rules/src/main/resources/example.rules.kts

```diff
@@ -264,6 +264,14 @@ fun RuleSet.deprecatedScopeExcludeReasonInOrtYmlRule() = ortResultRule("DEPRECAT
     }
 }
 
+fun RuleSet.missingContributingFileRule() = ortResultRule("MISSING_CONTRIBUTING_FILE") {
+    require {
+        -sourceTreeHasFile("CONTRIBUTING.md")
+    }
+
+    error("The project's code repository does not contain the file 'CONTRIBUTING.md' in the root directory.")
```

Comment:
Maybe just "... contain the file './CONTRIBUTING.md'." (similar in a later commit)?